tests/test_astronomer_registry_configmap.py

```diff
@@ -24,3 +24,24 @@ def test_astronomer_registry_configmap(kube_version):
     assert doc["kind"] == "ConfigMap"
     assert doc["apiVersion"] == "v1"
     assert bool(re.match("[+]?\\d+s", timeout))
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+def test_astronomer_registry_redirect(kube_version):
```

Comment:
does it make sense to add alternative test?

Reply:
@andriisoldatenko what are you proposing? There is only one other test case in this file, so it seems to me that having this test case that tests `registry.redirect.disable=True` makes sense.

Reply:
@neel-astro I think maybe what @andriisoldatenko is suggesting is we add a test with registry redirect enabled, but also disabled. Usually in a case like this we'll add the assertion for the default value in a more conglomerate test with a bunch of other defaults. So maybe add add the `partsed_config_yml` bit and assert False in the `test_astronomer_registry_configmap` test, which should probably be renamed to `test_astronomer_registry_configmap_defaults` or something.